docs/Maintainers.md

```diff
@@ -0,0 +1,95 @@
+Community Maintenance Committee
+===============================
+
+This document describes the Community Maintenance Committee of Open Enclave. By
+our liberal contribution policy outlined in our
+[governance model](GovernanceModel.md), maintainers are committers that are
+trusted to grant new committer rights, and grant new membership into the
+Committee.
+
+When making decisions, the Community Maintenance Committee uses a "consensus
+seeking" process. This means that most decisions should be reached by consensus,
+but when that fails, the Committee calls for a vote where the super majority
+(two-thirds) wins. This is to prevent obstructionism by removing the possibility
+of a one person veto.
+
+Committee Members
+-----------------
+
+| Name                 | Company   | Email                         | GitHub Alias   |
+|----------------------|-----------|-------------------------------|----------------|
+| Akash Gupta          | Microsoft | akagup@microsoft.com          | gupta-ak       |
+| Amaury Chamayou      | Microsoft | amaury.chamayou@microsoft.com | achamayou      |
+| Andrew Schwartzmeyer | Microsoft | andschwa@microsoft.com        | andschwa       |
+| Dave Thaler          | Microsoft | dthaler@microsoft.com         | dthaler        |
+| John Kordich         | Microsoft | johnkord@microsoft.com        | johnkord       |
+| Mike Brasher         | Microsoft | mikbras@microsoft.com         | mikbras        |
+| Simon Leet           | Microsoft | simon.leet@microsoft.com      | CodeMonkeyLeet |
+
+Committee Responsibilities
+--------------------------
+
+The primary responsibility of the Committee is to grant new committer rights
+(that is, write access to the main Open Enclave SDK repository or related
+repositories), and to grant new membership into the committee. Conversely, the
+Committee must also remove committer rights and membership from those found to
+be violating the project's Code of Conduct or otherwise negatively affecting the
+project's community health.
+
+This Committee is not intended to make every technical decision, as those should
+generally be made by agreement among committers as PRs are reviewed and merged.
```

Comment:
My experience with open source projects like Iotivity and AllJoyn is that a flat committee is a bad way to make technical decisions.  Instead, from experience it's better to assign a technical lead for each area, and they get to make the decision for that area, without having to wait for getting agreement from everyone else for every technical decision.   Policies/architecture for that area can be approved by the committee as the text states, but day to day technical decisions should be delegated to area leads if you want to maintain any degree of agility.

Reply:
That's why it's not making (most) technical decisions! We could later assign technical leads if we find it necessary, but we haven't had a need to do so yet.